app/styles/partials/welcome.scss

```diff
@@ -21,6 +21,8 @@
     .button {
       margin-top: 20px;
       font-size: 16px;
+      text-transform: uppercase;
```

Comment:
this will apply to any generic item and blue button, Please wrap it so it is specific to the navbar